src/Http/Http.Extensions/gen/StaticRouteHandlerModel/EndpointParameter.cs

```diff
@@ -130,35 +130,36 @@ private bool TryGetParsability(IParameterSymbol parameter, WellKnownTypes wellKn
 
         if (IsOptional)
         {
-            parsingBlockEmitter = (inputArgument, outputArgument) => $$"""
-                        {{parameter.Type.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)}} {{outputArgument}} = default;
-                        if ({{preferredTryParseInvocation(inputArgument, $"{inputArgument}_parsed_non_nullable")}})
-                        {
-                            {{outputArgument}} = {{$"{inputArgument}_parsed_non_nullable"}};
-                        }
-                        else if (string.IsNullOrEmpty({{inputArgument}}))
-                        {
-                            {{outputArgument}} = null;
-                        }
-                        else
-                        {
-                            wasParamCheckFailure = true;
-                        }
-""";
+            parsingBlockEmitter = (writer, inputArgument, outputArgument) =>
+            {
+                writer.WriteLine($$"""{{parameter.Type.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)}} {{outputArgument}} = default;""");
```

Comment:
```suggestion
                writer.WriteLine($"""{parameter.Type.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)} {outputArgument} = default;""");
```

Since this string and others contain no curly braces that need to be escaped.